packages/wifi_info_flutter/wifi_info_flutter/android/src/main/java/io/flutter/plugins/wifi_info_flutter/WifiInfoFlutterMethodChannelHandler.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.wifi_info_flutter;
+
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+
+/**
+ * The handler receives {@link MethodCall}s from the UIThread, gets the related information from
+ * a @{@link WifiInfoFlutter}, and then send the result back to the UIThread through the {@link
+ * MethodChannel.Result}.
+ */
+class WifiInfoFlutterMethodChannelHandler implements MethodChannel.MethodCallHandler {
+  private WifiInfoFlutter wifiInfoFlutter;
+
+  /**
+   * Construct the ConnectivityMethodChannelHandler with a {@code connectivity}. The {@code
```

Comment:
@bparrishMines It must be updated to Construct the WifiInfoFlutterMethodChannelHandle...